packages/solo/src/start.js

```diff
@@ -88,6 +89,14 @@ const atomicReplaceFile = async (filename, contents) => {
   }
 };
 
+const neverStop = () => {
+  return harden({
+    vatCreated: () => true,
+    crankComplete: () => true,
+    crankFailed: () => true,
+  });
```

Comment:
I added an `emptyCrank` to the policy since this PR was started

Reply:
Thanks for the heads-up.  The run policy type doesn't appear to be exposed outside SwingSet.

Reply:
Oh, found it... ambient `RunPolicy`.

Reply:
Yeah it's currently ambient :(